.github/workflows/actions.yaml

```diff
@@ -0,0 +1,22 @@
+name: CI
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: ["ubuntu-latest", "macos-latest"]
+        opa: ["v0.24.0"]
+    name: ${{ matrix.os }} opa ${{ matrix.opa }}
+    steps:
+      - uses: actions/checkout@v2
+      - run:
+        binary=$([[ "$OSTYPE" == "darwin"* ]] && echo "opa_darwin_amd64" || echo "opa_linux_amd64")
```

Comment:
these needs to be indented

Reply:
they are indented 2-space ... need more ?
... maybe simpler to suggest the formatting you want to see

Reply:
pushed to your branch. just for future reference, you can change `on` to be `on: [push, pull_request]` for testing purposes and test it on your branch.